src/main/resources/assets/styles/_upcomingReleases.scss

```diff
@@ -29,16 +36,19 @@
     }
   }
 
+  @include media-breakpoint-down(md) {
+    .calendar-day {
```

Comment:
Could this .calendar-day section be moved to line 23 in order to avoid having duplicate code (lines 16 and 39)? Or is there a reason for the duplication that I'm not seeing here?